app/cdap/api/pipeline.js

```diff
@@ -80,4 +80,10 @@ export const MyPipelineApi = {
   ),
 
   getAppVersions: apiCreator(dataSrc, 'GET', 'REQUEST', `${basepath}/versions`),
+  fetchAllPluginsProperties: apiCreator(
```

Comment:
doesn't this already exist somewhere else?

Reply:
I could not find another function that I could use, so I added it into the pipeline api.

Reply:
Does https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/api/replicator.js#L35 do the same thing?

Reply:
Yes it does, thanks! Updated the code.